ingest-app/src/cmr/ingest/services/jobs.clj

```diff
@@ -216,6 +221,14 @@
        context
        (ingest-events/provider-autocomplete-suggestion-reindexing-event provider)))))
 
+(defn update-completed-bulk-granule-task-statuses
```

Comment:
This function is exactly the same as what is in `update-completed-granule-task-statuses` function in the api bulk.clj file. Move this function to `granule-bulk-update-service`, and call it from this namespace and the api bulk namespace.